tests/e2e/createWallets.spec.ts

```diff
@@ -8,10 +8,10 @@ const { getByText, getByLabelText } = queries;
 
 const commonCreateWalletUserCreate = async () => {
   const user = USER.SINGLE();
-  const { page: welcomePage, browser } = await loadExtension();
+  const { page: page, browser } = await loadExtension();
```

Comment:
Renaming `welcomePage` to `page` because it's the general puppeteer page object which can change during a test.